os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/src/rtl8721d_bt_calibration.c

```diff
@@ -279,13 +279,13 @@ boolean _btiqk_lok_8721d(void *dm_void, u32 *u32bt_iqk_x, u32 *u32bt_iqk_y, u32
 	{
 		btiqk_success |= 0x01;
 #if DBG_BT_CALIBRATION 
-        DBG_8195A("the IQK and LOK procedure is done");	/* success != workable */
+        printf("the IQK and LOK procedure is done");	/* success != workable */

```

Comment:
Need line ending at the end of the message for printf.

Reply:
Added